dev-utils/ci-tav-slow-jobs.sh

```diff
@@ -19,6 +19,12 @@ fi
 set -o errexit
 set -o pipefail
 
+# Make sure we have json package
+jsonPath=$(which json)
```

Comment:
Note to reviewer: sometimes I find myself switching to a different node version and the `json` package is not present.